brave/brave-ios
# Fix #4042: Copy and Logic update for universal app open

Changing universal link open value to open in brave with text and migration



## Summary of Changes


This pull request fixes #4042

## Submitter Checklist:


## Test Plan:

- Navigate Brave Shields Privacy Settings and check the text is update and valuer is reversed 
- Check the new value is working accordding to toggle

## Screenshots:

![IMG_0877](https://user-images.githubusercontent.com/6643505/185482472-bc87a2fb-3f33-4e8c-9969-c8b4f9adab0c.PNG)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
